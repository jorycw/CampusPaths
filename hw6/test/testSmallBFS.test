# Test the parser on a graph that contains several nodes
# with two edges each. This is a simple starting test and only
# tests regular cases before testing scalability
# NOT A TRUE UNIT TEST

LoadGraph heroes staffSuperheroes.tsv

FindPath heroes Ernst-the-Bicycling-Wizard Perkins-the-Magical-Singing-Instructor

FindPath heroes Grossman-the-Youngest-of-them-all Notkin-of-the-Superhuman-Beard

FindPath heroes NotIn ThisGraph