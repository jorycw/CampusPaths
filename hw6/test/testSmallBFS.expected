# Test the parser on a graph that contains several nodes
# with two edges each. This is a simple starting test and only
# tests regular cases before testing scalability
# NOT A TRUE UNIT TEST

loaded graph heroes

path from Ernst-the-Bicycling-Wizard to Perkins-the-Magical-Singing-Instructor:
Ernst-the-Bicycling-Wizard to Perkins-the-Magical-Singing-Instructor via CSE331

path from Grossman-the-Youngest-of-them-all to Notkin-of-the-Superhuman-Beard:
Grossman-the-Youngest-of-them-all to Notkin-of-the-Superhuman-Beard via CSE331

unknown character NotIn
unknown character ThisGraph
